### DISCUSSION TITLE
Custom Error Messages

### DISCUSSION THREAD

**Discussion by a-znk** [wevm/wagmi#1690] [Q&A]
Working through displaying errors messages to users on my Dapp, and I'm running into some problems displaying them in a digestible way... Simply displaying `Error: {(prepareError || error)?.message}` as suggested in the Wagmi example does not give a great user experience. 

I've been experimenting with `error.reason` however this is creating a type error Property 'reason' does not exist on type 'Error'

Does anyone have a good solution for displaying error messages to users? Maybe even custom messages? like "You don't have enough Eth in your wallet to buy this NFT!" or something

**Comment by wajidsiddiqi** [comment]
Hi there! Have you found the solution?
